ghdocs/Tutorial.md

```diff
@@ -0,0 +1,153 @@
+## Overview
+In this tutorial, let's build a simple react app with a DocumentCard fabric component.
```

Comment:
I'd consider rewording to something akin to: "...let's build a simple React app with a Fabric React DocumentCard.